apps/payments/next/app/layout.tsx

```diff
@@ -18,7 +18,9 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body>
-        <main className="mt-16 min-h-[calc(100vh_-_4rem)]">{children}</main>
+        <main className="mt-16 tablet:mt-20 min-h-[calc(100vh_-_4rem)]">
```

Comment:
`tablet:mt-18`, 20 caused a small gap to appear on Checkout.

Reply:
This seems to be an issue with the checkout/upgrade layout, so I'm fixing it on that side.